### PR TITLE
redis client

### PR DESCRIPTION
**Pull request by coyle**

### CODE REVIEW

File: storage/redis/overlay.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	if err != nil {
+		return nil, err
+	}
+
+	o := &OverlayClient{
+		DB: rc,
+	}
+
+	// ping here to verify we are able to connect to the redis instacne with the initialized client.
+	if err := o.DB.Ping(); err != nil {
```
[the remark this answers is not in the document]
Comment (coyle) [reply storage/redis/overlay.go]: good point. removed

File: storage/redis/client.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package redis
```
[the remark this answers is not in the document]
Comment (coyle) [reply storage/redis/client.go]: I'll write a script and add it to the make lint step. Not sure of specific linter that does this out of the box.

[the remark this answers is not in the document]
Comment (nfarah86) [reply storage/redis/client.go]: LGTM - talked to @coyle already

File: storage/redis/overlay_test.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package redis
+
+import (
+	"testing"
+
+	"github.com/coyle/storj/protos/overlay"
```
Comment (jtolds) [inline storage/redis/overlay_test.go]: github.com/coyle/storj?

**Pull request merged**